cluster/manifests/roles/poweruser-role.yaml

```diff
@@ -58,19 +58,6 @@ rules:
   - services/proxy
   verbs:
   - get
-- apiGroups:
-  - ''
-  resources:
-  - secrets
-  verbs:
-  - create
-  - delete
-  - deletecollection
-  - get
-  - list
-  - patch
-  - update
-  - watch
```

Comment:
Write permissions are already specified for `secrets` in the rule above, here they are redundant, so they're dropped alongside read permissions.